src/modules/app_commands/sendPipelineRequest.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * App command to send pipeline request.
+ *
+ * @param {string} pipelineName Name of the pipeline.
+ * @param {string} pipelineInput Input parameters for the pipeline.
+ * @returns {AppCommand}
+ */
+export function sendPipelineRequest(pipelineName, pipelineInput = {}) {
+  return {
+    c: 'sendPipelineRequest',
+    p: {
+      serial: pipelineName,
+      name: pipelineName,
+      input: pipelineInput,
+    },
+  };
+}
+
+/**
+ * App command to send trusted pipeline request.
+ *
+ * @param {string} pipelineName Name of the pipeline.
+ * @param {string} pipelineInput Input parameters for the pipeline.
```

Comment:
here also. You can also write the optional value (and the default value) in the jsdoc: [pipelineInput={}]

Reply:
Done